variables.ini.dist

```diff
@@ -122,6 +122,7 @@ OPENSTACK_DEFAULT_ROUTER = ; openstack_router
 OPENSTACK_USER_ROLE = ; defaultMemberRole
 OPENSTACK_ADMIN_ROLE = ; admin
 ATMOSPHERE_SUPERUSER = ; atmo_superuser
+ATMOSPHERE_SECRET_SEED = ; (Random value:) 9oe5q7cp714zmmjl51
```

Comment:
Would we benefit by describing how this is created or generated? Is it just an SHA1 fingerprint? Is there any expected length to the sequence?